src/Akka.HealthCheck/Transports/LivenessTransportActor.cs

```diff
@@ -19,50 +22,76 @@ namespace Akka.HealthCheck.Transports
     public sealed class LivenessTransportActor : ReceiveActor
     {
         private const int LivenessTimeout = 1000;
-        private readonly IActorRef _livenessProbe;
+        private readonly List<IActorRef> _livenessProbes;
         private readonly ILoggingAdapter _log = Context.GetLogger();
         private readonly IStatusTransport _statusTransport;
         private readonly bool _logInfo;
 
-        public LivenessTransportActor(IStatusTransport statusTransport, IActorRef livenessProbe, bool log)
+        public LivenessTransportActor(IStatusTransport statusTransport, ImmutableDictionary<string, IActorRef> livenessProbes, bool log)
         {
             _statusTransport = statusTransport;
-            _livenessProbe = livenessProbe;
+            var probeReverseLookup = livenessProbes.ToImmutableDictionary(kvp => kvp.Value, kvp => kvp.Key);
+            _livenessProbes = livenessProbes.Values.ToList();
             _logInfo = log;
 
             ReceiveAsync<LivenessStatus>(async status =>
             {
+                var probeName = probeReverseLookup[Sender];
+                
                 if (_logInfo)
-                 _log.Info("Received liveness status. Live: {0}, Message: {1}", status.IsLive, status.StatusMessage);
+                 _log.Info("Received liveness status from probe [{0}]. Live: {1}, Message: {2}", probeName, 
+                     status.IsLive, status.StatusMessage);
                
                 var cts = new CancellationTokenSource(LivenessTimeout);
                 TransportWriteStatus writeStatus = null;
-                if (status.IsLive)
-                    writeStatus = await _statusTransport.Go(status.StatusMessage, cts.Token);
-                else
-                    writeStatus = await _statusTransport.Stop(status.StatusMessage, cts.Token);
+                try
+                {
+                    if (status.IsLive)
+                        writeStatus = await _statusTransport.Go($"[{probeName}] {status.StatusMessage}", cts.Token);
```

Comment:
Tagged the status message with the probe name. Will this be good enough formatting wise?